tests/kernel_performance/hanaperf_postinstall.pm

```diff
@@ -0,0 +1,153 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Post installation for HANA perf SLE16 testing
+# Inherit from install_qatestset.pm
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+package hanaperf_postinstall;
+use base 'y2_installbase';
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+use utils;
+use testapi;
+use Utils::Architectures;
+use repo_tools 'add_qa_head_repo';
+use mmapi 'wait_for_children';
+use version_utils qw(is_sle);
+use ipmi_backend_utils;
+
+sub install_sleperf {
+    my $sleperf_source = get_var('SLE_SOURCE');
+    my $ver_path = "/root";
+
+    # Download SLEperf package, extract and install
+    assert_script_run("wget --quiet -O $ver_path/sleperf.tar $sleperf_source 2>&1");
+    assert_script_run("tar xf /root/sleperf.tar -C /root");
+    assert_script_run("cd /root/sleperf/SLEPerf; ./installer.sh scheduler-service");
+    assert_script_run("cd /root/sleperf/SLEPerf; ./installer.sh common-infra");
+}
+
+sub extract_settings_qaset_config {
+    my $values = shift;
+    my @fields = split(/;/, $values);
+    if (scalar @fields > 0) {
```

Comment:
I don't think the `scalar` keyword is needed. Perl will turn the list to a scalar by the `>` operator:
```
root@mango:/var/lib/openqa/tests/sle [master|…12] > perl
my @l = qw(hola Dawei);
print "More than 0 elements: [@l]\n" if (@l > 0);
@l = ();
print "More than 0 elements: [@l]\n" if (@l > 0);
More than 0 elements: [hola Dawei]
```

Reply:
thanks! fixed.